src/components/UI/Text/Textarea.vue

```diff
@@ -36,26 +39,30 @@
       >
         <inline-svg :src="startIconObj"></inline-svg>
       </div>
-
       <!-- input text area -->
-      <textarea
-        class="p-2 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
+
+      <div
+        contenteditable="true"
```

Comment:
This was added to make div contenteditable. But yes since ql-editor is the child component and is editable ,this was not necessary.I will remove this.